packages/@okta/vuepress-site/docs/guides/style-the-widget/before-you-begin/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Before you begin
+---
+
+The Okta Sign-In Widget is a JavaScript library that gives you a fully-featured and customizable sign-in experience that can be used to authenticate users on any website. How you customize the Widget depends on whether Okta is hosting it or you are embedding it in your app (self-hosting the Widget).
+
+* Self-hosted Sign-in Widget: After you have [installed the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget#getting-started) into your project and configured the authentication scenarios that you want to support, you then want to customize. You can apply customizations to match your branding using CSS and JavaScript. See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget).
```

Comment:
"you then want to customize"  suggested edit:  "you can can then customize"

Reply:
"See [Styling the self-hosted Sign-In Widget](#styling-the-self-hosted-sign-in-widget)." - Is the URL complete?

Reply:
holy cow, Moti - thank you. TOTALLY missed that.